content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,84 @@ title: What is an **attribute**?
+
+# Akeneo attribute types
+
+Here are the attribute types available in Akeneo:
+- **identifier** attribute: a code to identify your product, this code must be unique. It can be a SKU, a MPN... This attribute is mandatory.
+- **text** attribute: a single-line text field that can contain **up to 255 characters**, it is usually used for a product name.
+- **text area** attribute: a multi-line text field that can be used for a product description.
+- **simple** attribute: a single-choice select field coming with custom options. Only one value can be selected amongst the available options. 
+- **multi select** attribute: a multi-choice select field coming with custom options. More than one value can be selected amongst the available options.
+- **boolean** attribute: a yes/no attribute
+- **date** attribute: a date field, the PIM will display a calendar to choose the date, which includes day, month and year.
+- **number** attribute: a single-line field that can only contain digits. 
+- **metric** attribute: a single-line field composed of a 1st field containing a number and a second field containing a metric unit. It allows you to automatically convert metric values into others to fit your export needs. You can check the [Manage your Channels](manage-your-channels.html#create-a-channel) article to have more information about this feature.
+- **price** attribute: a price attribute, displayed values will depend on the currencies enabled in the PIM. Learn [here](manage-your-currencies.html#enabledisable-a-currency) how to enable or disable a currency in Akeneo PIM.
+- **image** attribute: a drag and drop down zone to upload an image (extensions allowed: gif, jfif, jif, jpeg, jpg, pdf, png, psd, tif, tiff)
+- **file** attribute: a drag and drop down zone to upload a file (extensions allowed: csv, doc, docx, mp3, pdf)
+- **asset collection** attribute _(Entreprise Edition only)_: an advanced attribute type to manage several digital resources like a video, picture, pdf file... Read the following article [What is an asset?](what-is-an-asset.html) to learn more about this attribute type
+- **reference entity simple link** attribute _(Entreprise Edition only)_: allows enriching common data related to products with a rich content (text, images...), more complex than just a code and labels. To learn more about Reference entities, read our article [What is a Reference Entity?](what-about-reference-entities.html)
+- **reference entity multiple links** attribute _(Entreprise Edition only)_: the same as above but allowing you to manage multiple links
+- **reference data simple select** attribute: allows you to manage any kind of data that has its own properties, as a single-choice select. 
+- **reference data multi select** attribute: allows you to manage any kind of data that has its own properties, as a multi-choice select. 
+
```

Comment:
Maybe adding also the fact that we advise the use of reference entities for users of Version 3.0+
& explaining that reference data belongs to the CustomEntityBundle. 
We had an SDS ticket about that recently https://akeneo.atlassian.net/browse/SDS-10605

Reply:
I could add something like: 
The use of reference data implies a custom development (using the CEB) whereas reference entities are natively integrated into Akeneo PIM. If you are starting a new project, it's better to use the Reference entities.
Moreover the "CSVToReferenceEntity" tool is available to migrate data from the CustomEntityBundle to the Reference Entities records, via a CSV file using the API. It can also be used to initialise reference entities data from an ERP/MDM.

Reply:
it's better to use the Reference entities > we strongly advise you to... to me it's better is a bit too soft, as we won't make support on the CEB.

Reply:
edited! I've added more info about ref data and ref entities! If you can check pls :)